circle.yml

```diff
@@ -18,3 +22,8 @@ test:
     - eval `opam config env` && make all
     - make test
     - eval `opam config env` && make test-qcow
+    # Go related steps
+    - mkdir -p $(dirname "$PROJECT_ROOT")
+    - ln -s $(pwd) "$PROJECT_ROOT"
+    - go get github.com/golang/lint/golint
```

Comment:
Move it into the Makefile in a `setup` rule perhaps? More convenient for non-CI use there.